### PR TITLE
DO NOT MERGE - Just for comments

### PR DESCRIPTION
**Pull request by irsal**

### CODE REVIEW

File: node/src/chain_spec/oak.rs [hunk 1]
```diff
@@ -19,9 +19,9 @@ use primitives::{AccountId, AuraId, Balance};
...
 const TOKEN_SYMBOL: &str = "OAK";
 const SS_58_FORMAT: u32 = 51;
-static RELAY_CHAIN: &str = "rococo-local";
+static RELAY_CHAIN: &str = "polkadot";
 static STAGING_RELAY_CHAIN: &str = "rococo";
-const REGISTERED_PARA_ID: u32 = 2114;
+const REGISTERED_PARA_ID: u32 = 2090;
 const REGISTERED_STAGING_PARA_ID: u32 = 4103;
 
 /// The default XCM version to set in genesis config.
```
Comment (irsal) [inline node/src/chain_spec/oak.rs:27]: need to add oak_live()

File: runtime/oak/src/lib.rs [hunk 2]
```diff
@@ -305,7 +305,7 @@ impl frame_system::Config for Runtime {
 
 parameter_types! {
 	// One storage item; key size is 32; value is size 4+4+16+32 bytes = 56 bytes.
-	pub const DepositBase: Balance = deposit(1, 88);
+	pub const DepositBase: Balance = deposit(1, 88); // IRSAL: but this says 88? Intentional?
```
Comment (irsal) [inline runtime/oak/src/lib.rs:308]: @rhuttman - any ideas on this? Charles implemented multi-sig, but i wanted to make sure this makes sense to you

**Comment by irsal** [comment]
- [ ] XCM configs https://github.com/OAK-Foundation/OAK-blockchain/wiki/HRMP-Channels-and-You
- [ ] Keeping 0.01 OAK for ED

File: runtime/oak/src/lib.rs [hunk 3]
```diff
@@ -1040,15 +1040,15 @@ pub struct ClosedCallFilter;
...
-			Call::Balances(_) => false,
-			Call::Bounties(_) => false,
-			Call::Currencies(_) => false,
-			Call::ParachainStaking(_) => false,
-			Call::PolkadotXcm(_) => false,
-			Call::Treasury(_) => false,
-			Call::XTokens(_) => false,
-			Call::AutomationPrice(_) => false,
+			Call::AutomationTime(_) => true,
+			Call::Balances(_) => true,
```
Comment (irsal) [inline runtime/oak/src/lib.rs:1044]: Keeping 10 decimlas for OAK. Is there an issue w/ KSM being 12 decimals and DOT being 10 decimals. @justinzhou93 to double check

File: runtime/oak/src/lib.rs (hunk 2 above)
Comment (rhuttman) [reply runtime/oak/src/lib.rs:308]: this was lifted straight from the polkadot runtime so should be fine

File: runtime/oak/src/xcm_config.rs [hunk 4]
```diff
@@ -181,7 +181,7 @@ pub type Barrier = DenyThenTry<
 
 /// Based on the precedent set by other projects. This will need to be changed.
 pub fn dot_per_second() -> u128 {
-	CurrencyId::DOT.cent() * 16
+	CurrencyId::DOT.cent() * 16 // IRSAL: Laura's going to cover this with her PR
```
Comment (rhuttman) [inline runtime/oak/src/xcm_config.rs:184]: her PR moves these values to a pallet, but we still need the right values for them

File: runtime/oak/src/lib.rs (hunk 3 above)
Comment (rhuttman) [reply runtime/oak/src/lib.rs:1044]: Just changes the token conversion rates.

File: runtime/oak/src/lib.rs [hunk 5]
```diff
@@ -1040,15 +1040,15 @@ pub struct ClosedCallFilter;
...
-			Call::AutomationTime(_) => false,
-			Call::Balances(_) => false,
-			Call::Bounties(_) => false,
-			Call::Currencies(_) => false,
-			Call::ParachainStaking(_) => false,
-			Call::PolkadotXcm(_) => false,
-			Call::Treasury(_) => false,
-			Call::XTokens(_) => false,
-			Call::AutomationPrice(_) => false,
+			Call::AutomationTime(_) => true,
```
Comment (rhuttman) [inline runtime/oak/src/lib.rs:1043]: why did you swap these to `true`?

Comment (irsal) [reply runtime/oak/src/lib.rs:1043]: isn't this the valve section?

File: runtime/oak/src/xcm_config.rs [hunk 6]
```diff
@@ -197,7 +197,7 @@ parameter_types! {
 			1,
 			X1(Parachain(parachains::testchain::ID)),
 		).into(),
-		oak_per_second() * 2
+		oak_per_second() * 2 // IRSAL: why?
```
[the remark this answers is not in the document]
Comment (irsal) [reply runtime/oak/src/xcm_config.rs:200]: ok

File: runtime/oak/src/lib.rs (hunk 5 above)
Comment (rhuttman) [reply runtime/oak/src/lib.rs:1043]: It is for the valve panic button. Anything in here that returns false will be filtered when we turn valve off completely. This is separate and overrides the gates.

**Pull request closed without merging**